src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -190,6 +177,24 @@ public bool TryFetch(OptionKey optionKey, out object value)
             return true;
         }
 
+        private bool DeserializeCodeStyleOption<T>(ref object value)
+        {
+            if (value is string serializedValue)
+            {
+                try
+                {
+                    value = CodeStyleOption<T>.FromXElement(XElement.Parse(serializedValue));
+                    return true;
+                }
+                catch (Exception)
```

Comment:
We do.  ```CodeStyleOption<TEnum>``` Will migrate ```CodeStyleOption<bool>``` so that 'false' is mapped to the 0 value of TEnum and true is mapped to the 1 value.   This likely needs a lot of documentation (and tests to validate that this works properly.